### ISSUE TITLE
Consider using pkg_resource to locate data files, rather than environment variable

### ISSUE THREAD

**Issue by dstndstn** [legacysurvey/unwise_psf#4]
eg,
```
    from pkg_resources import resource_filename
    filename = resource_filename('unwise_psf', 'etc/psf_model_w1_neo4_unwisecat.fits')
```

**Comment by weaverba137** [comment]
I agree with this, but it takes *slightly* more setup than that.  Unfortunately, the py/ directory is not set up as a proper Python package.  So that has to happen first.  Then move etc/ to py/unwise_psf/data, then you can do:
```python
filename = resource_filename('unwise_psf', 'data/psf_model_w1_neo4_unwisecat.fits')
```
The data directory also needs to be declared in the setup.py file so that the data are bundled with the code.  There are a number of examples of this in desihub.

**Comment by dstndstn** [comment]
Ah yes, thanks for the correction.

**Comment by moustakas** [comment]
@ameisner @schlafly  Any thoughts on implementing this proposed change for DR9?  It would definitely help with the bookkeeping and provenance within the pipeline.

**Comment by ameisner** [comment]
@moustakas @dstndstn @schlafly 

I tried to address this with pull request #5.

**Comment by ameisner** [comment]
I just merged #5 and created a "dr9.0" tag for this updated version of unwise_psf.
